### PR TITLE
Fix document reordering

### PR DESCRIPTION
**Pull request by djnunez-aot**
*Issue #1001 :*
https://github.com/bcgov/met-public/issues/1001

Expectation:
-When document folders or files are edited do not reorder the data files randomly.

Current version: 
-When documents are reordered they randomly shuffle order.

This pr:
-Sort folders by ascending id order to  make document order consistent in the widget tab 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the met-public license (Apache 2.0).

### CODE REVIEW

**Review by saravanpa-aot — CHANGES_REQUESTED** [review]
Can you improve the PR description to state the problem that you are handling ?
Or add a test to reflect these changes?
Without both of these , its hard to know what exactly this change does

File: met-api/src/met_api/services/widget_documents_service.py [hunk 1]
```diff
@@ -29,6 +29,7 @@ def get_documents_by_widget_id(widget_id):
     @staticmethod
     def _attach_file_nodes(docs, root):
         files = list(filter(lambda doc: doc.type == DocumentType.FILE.value, docs))
+        files.sort(key=lambda doc: doc.id)
```
Comment (jadmsaadaot) [inline met-api/src/met_api/services/widget_documents_service.py:32]: is there a requirement to sort by id?

**Review by tiagograf-AOT — APPROVED** [review]

File: met-api/src/met_api/services/widget_documents_service.py (hunk 1 above)
Comment (djnunez-aot) [reply met-api/src/met_api/services/widget_documents_service.py:32]: yes without sorting by id for the files, when there is multiple files without a folder they randomly change order after being edited.

**Review by jadmsaadaot — APPROVED** [review]

File: met-api/tests/unit/models/test_widget_documents.py [hunk 2]
```diff
@@ -55,6 +55,7 @@ def test_get_all_by_widget_id(session):
     session.commit()
 
     expected_docs = WidgetDocumentsModel.get_all_by_widget_id(widget.id)
+    all(expected_docs[i].id <= expected_docs[i + 1].id for i in range(len(expected_docs) - 1))
```
Comment (saravanpa-aot) [inline met-api/tests/unit/models/test_widget_documents.py:58]: what does this do?

Comment (djnunez-aot) [reply met-api/tests/unit/models/test_widget_documents.py:58]: it checks if the expected docs are order in ascending order

**Review by saravanpa-aot — APPROVED** [review]

**Pull request merged**